manifests/s/Salesforce/sfdx-cli/7.190.2.0/Salesforce.sfdx-cli.installer.yaml

```
@@ -0,0 +1,13 @@
+# Automatically updated by the winget bot at 2023/Mar/03
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Salesforce.sfdx-cli
+PackageVersion: 7.190.2.0

```

```suggestion
PackageVersion: 7.190.2
```
Version needs fixing